docs/guides/configuration-guide/loadbalancer.md

```diff
@@ -25,42 +25,69 @@ hosts_additional_entries:
 
 ## TLS certificates
 
-To enable external TLS encryption:
+## General procedure
 
-```yaml title="environments/kolla/configuration.yml"
-kolla_enable_tls_external: "yes"
-```
+:::warning
+It is very important that important decisions are made at the beginning in connection with
+TLS encryption and domains, as these decisions are very difficult to change at a later
+date or can only be changed at great expense.
```

Comment:
I think this is a good warning, but we should follow up with which decisions exactly are important and why they can't be changed easily later on.

Without this information the reader is a little bit lost imho.

Reply:
You're right, the sentence is very generic.
I've tried to describe it a little differently - maybe it's better this way.